Corona-Warn-App/src/main/java/de/rki/coronawarnapp/service/submission/SubmissionService.kt

```diff
@@ -88,7 +88,6 @@ object SubmissionService {
 
     fun submissionSuccessful() {
         BackgroundWorkScheduler.stopWorkScheduler()
-        LocalData.numberOfSuccessfulSubmissions(1)
```

Comment:
Is this not supposed to be called after a successfull submission of keys?

Reply:
that's correct, it's been rectified now.